### PR TITLE
loading indicators everywhere content is loading;

### PR DESCRIPTION
**Pull request by Jeppes**

### CODE REVIEW

File: app/src/main/java/se/ths/kth/Aramda/components/armada/ArmadaPresenter.java [hunk 1]
```diff
@@ -32,11 +32,16 @@ public void start() {
...
+        view.startLoadingIndicator();
         dataSource.getPartners().subscribe(this::onPartnersLoaded);
     }
 
     private void onPartnersLoaded(List<Partner> partners) {
         view.setPartners(partners);
+
+        if (!partners.isEmpty()) {
+            view.stopLoadingIndicator();
+        }
```
Comment (apals) [inline app/src/main/java/se/ths/kth/Aramda/components/armada/ArmadaPresenter.java]: why do we need to check if it is empty

Comment (Jeppes) [inline app/src/main/java/se/ths/kth/Aramda/components/armada/ArmadaPresenter.java]: The first emission is always an empty realm list

Comment (apals) [inline app/src/main/java/se/ths/kth/Aramda/components/armada/ArmadaPresenter.java]: what if we have no partners

Comment (Jeppes) [inline app/src/main/java/se/ths/kth/Aramda/components/armada/ArmadaPresenter.java]: then they will never load

Comment (Jeppes) [inline app/src/main/java/se/ths/kth/Aramda/components/armada/ArmadaPresenter.java]: There is no way of knowing in that case

**Pull request merged**